programming basics/Python Built-ins.ipynb

```diff
@@ -0,0 +1,727 @@
+{
```

Comment:
<p>I suggest putting in a bit of info here about how Python is dynamically typed...although you can force it to do as you like (amend the code below to reflect that, I put in a suggestion).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PTC-CMC/cybercamp/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>